app/controllers/course_summaries_controller.rb

```diff
@@ -72,6 +71,18 @@ def download_csv_grades_report
 
   private
 
+  def generate_out_of_row
+    # This function creates the second row of the grades summary, containing an assignments max mark where
```

Comment:
This comment could be worded better. After the comma, something like `containing the max mark of every assessment.` Make sure to end with a period.